.github/workflows/domain_ci.yml

```diff
@@ -46,7 +46,8 @@ jobs:
 
       - name: Extract torchvision ref
         id: torchvision
-        run: echo "::set-output name=ref::$(python -c 'import torchvision; print(torchvision.version.git_version)')"
+        run:
+          echo "name=ref::$(python -c 'import torchvision; print(torchvision.version.git_version)')" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "ref=$(python -c 'import torchvision; print(torchvision.version.git_version)')" >> $GITHUB_OUTPUT
```

Reply:
Thanks! Somehow I didn't clean this up thoroughly.